src/Whoops/Exception/Inspector.php

```diff
@@ -42,7 +42,11 @@ public function getException()
     }
 
     /**
+     * Use getClass.
+     *
      * @return string
+     *
+     * @deprecated
```

Comment:
When you deprecate a method you want to replace its body to `$this->newMethod()`, so that if we change the new method, the deprecated would include the new functionality.